src/features/world/scenes/BeachScene.ts

```diff
@@ -191,10 +190,8 @@ export class BeachScene extends BaseScene {
       this.add.sprite(320, 580, "locked_disc").setDepth(1000000000);
     }
 
-    const chest = this.add.sprite(320, 600, "wooden_chest");
-    chest.setInteractive({ cursor: "pointer" }).on("pointerdown", () => {
```

Comment:
I don't think we want to remove this interaction. I can't click on the wooden chest in the beach any more.